crowdsec-docs/docs/getting_started/install_source.mdx

```diff
@@ -0,0 +1,200 @@
+    defaultValue="DEB"
+    groupId="build_platform"
+    values={[
+      {label: 'DEB', value: 'DEB'},
+      {label: 'RPM', value: 'RPM'},
+      {label: 'Arch', value: 'Arch'},
+    ]}>
+    <TabItem value="DEB">
+      <CodeBlock className="language-bash">apt install gcc</CodeBlock>
+    </TabItem>
+    
+    <TabItem value="RPM">
+        <CodeBlock className="language-bash">dnf install gcc</CodeBlock>
+    </TabItem>
+    <TabItem value="Arch">
+      <CodeBlock className="language-bash">pacman -S gcc</CodeBlock>
+    </TabItem>
+  </Tabs>
+- pkg-config
+  <Tabs
+    defaultValue="DEB"
+    groupId="build_platform"
+    values={[
+      {label: 'DEB', value: 'DEB'},
+      {label: 'RPM', value: 'RPM'},
+      {label: 'Arch', value: 'Arch'},
+    ]}>
+    <TabItem value="DEB">
+      <CodeBlock className="language-bash">apt install pkg-config</CodeBlock>
+    </TabItem>
+    
+    <TabItem value="RPM">
+        <CodeBlock className="language-bash">dnf install pkg-config</CodeBlock>
+    </TabItem>
+    <TabItem value="Arch">
+      <CodeBlock className="language-bash">pacman -S pkg-config</CodeBlock>
+    </TabItem>
+  </Tabs>
+
+### Optional
+
+- RE2
+  <Tabs
+    defaultValue="DEB"
+    groupId="build_platform"
+    values={[
+      {label: 'DEB', value: 'DEB'},
+      {label: 'RPM', value: 'RPM'},
+      {label: 'Arch', value: 'Arch'},
+    ]}>
+    <TabItem value="DEB">
+      <CodeBlock className="language-bash">apt install libre2-dev g++</CodeBlock>
+    </TabItem>
+    
+    <TabItem value="RPM">
+        <CodeBlock className="language-bash">dnf install libre2-dev g++</CodeBlock>
+    </TabItem>
+    <TabItem value="Arch">
+      <CodeBlock className="language-bash">pacman -S re2 base-devel</CodeBlock>
+    </TabItem>
+  </Tabs>
+
+## Building
+
+### Clone
+
+```bash
+git clone https://github.com/crowdsecurity/crowdsec
+cd crowdsec
+```
+
+### Build
+
+```bash
+make [build_flags] build
+```
+
+#### Build flags
+
+##### Required
+
+- `BUILD_VERSION=v1.2.3` - The version you want to build. This is required for the build to work.
+
+:::info
+Do not use a version we have already released as you will get old hub parsers. We recommend using the latest tag from the [releases page](https://github.com/crowdsecurity/crowdsec/releases/latest).
+:::
+
+##### Optional
+
+:::info What is a static build?
+Static builds are builds that do not require any external dependencies to run. This means a compiled binary on your system will work on any other system running your architecture and linux/windows/freebsd. As an example if I compile a static build on my Arch Linux machine, I can copy that binary to a Debian machine and it will work without any issues. 
+:::
+
+- `BUILD_STATIC=1` - Build a static binary:
+  - DEB: none
+  - RPM/Arch:
+    - RPM: Enable [crb repo](https://linux.how2shout.com/enable-crb-code-ready-builder-powertools-in-almalinux-9/#:~:text=CRB%20(Code%20Ready%20Builder)%20is,their%20workflow%20and%20increase%20productivity.)
```

Comment:
as you can see from the vagrant files, the crb repo has different names for different folks: for oracle 9 it's "dnf config-manager --set-enabled ol9_codeready_builder" and so on. I think I caught them all but I'm pretty sure they'll keep changing names because they can :)